### ISSUE TITLE
API to export the `py3dmol.view` from a notebook as a separate interactive html file.

### ISSUE THREAD

**Issue by meneshail** [3dmol/3Dmol.js#678]
**Is your feature request related to a problem? Please describe.**
The `py3Dmol` is an excellent package to deliver interactive 3D molecules with customizable displaying options. It is sometimes necessary to export the interactive `py3dmol.view` separately for presentation usage, but there seems not an API to do the job.

**Describe the solution you'd like**
I'm not very familiar with javascript, but this seems to be quite feasible since there is no essential difference between exporting the whole notebook and exporting a single widget, in terms of technical difficulty.

**Additional context(Optional)**
Actually, as for `nglview`, another commonly-used molecule visualization package with a javascript backend, there is an API doing exactly the things I want.  And here are the links to their documentation and source code: [doc](http://nglviewer.org/nglview/latest/api.html#nglview.write_html), [src](http://nglviewer.org/nglview/latest/_modules/nglview/widget.html#write_html).
Thanks in advance:)

**Comment by dkoes** [comment]
Can you explain more what you want? py3dmol is designed so you can save a notebook as an html file and it will still work. It isn’t clear to me what functionality you want.

**Comment by meneshail** [comment]
Sorry not being clear enough. I want to export a specific `py3dmol.view` object as an HTML file, but not the whole notebook.

For example, I could do the following thing with `nglview`:
![demo_screenshot](https://github.com/3dmol/3Dmol.js/assets/75558787/1336bec1-3bf3-4370-8e8b-2b8383e5ba3d)


This will write the `view` widget to an HTML file without the other parts of the notebook. ([demo file link](https://drive.google.com/file/d/1I-kYHweDegxzh10HyzNJxq9nnS4nHgYq/view?usp=sharing))
![demo_screenshot2](https://github.com/3dmol/3Dmol.js/assets/75558787/77df38cf-5af6-451e-81fa-04da9d5f1b7b)

In short, sometimes I don't need the whole notebook with me, but just the interactive molecule view with customized representations and layouts.

**Comment by dkoes** [comment]
I've added `viewer.write_html` as a function in py3dmol 2.0.3 (just released).  You can provide a file name or a file object.  If nothing is provided it returns the html string.  There is also the option `fullpage` which if set to True will produce html that ignores the specified viewer size and creates a web page that is 100% viewer.

If you have to use an older version, you can also get the html string by calling `viewer._make_html` but then you need to write it to the file yourself.

Please verify this works as expected and, if so, close the issue.

**Comment by meneshail** [comment]
Thanks for your rapid update! I have tested the newly addec function (in '2.0.3') as well as saving via `viewer._make_html` with an older version ('2.0.1.post1'), and both work exactly as expected.
Closing the issue now.
